### PR TITLE
Added French language file, with translations.

### PR DESCRIPTION
**Pull request by Philip-Nicolas**

### CODE REVIEW

**Comment by suve** [comment]
Thanks for the PR. There are two issues I can see:
- In the `.po` file, "rms is happy" / "rms is disappoint" are not translated.
- Could you also translate the manpage (`man/en.man`)?

**Comment by Philip-Nicolas** [comment]
Yep. Thought the rms is happy parts were for the ascii stuff, will do.

File: lang/fr.po [hunk 1]
```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: fr\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Usage: vrms-rpm [options]"
```
Comment (suve) [inline lang/fr.po]: Is this string translated?

File: lang/fr.po [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_help_option_ascii"    
+msgstr "    Afficher l'art ASCII rms quand aucun progiciels non-gratuits sont trouvés,\n"
+       "    ou quand des progiciels non-gratuits comprennent 100%% ou plus du total."
+msgid "msg_help_option_explain"
+msgstr "    Afficher les licenses pour justifier la classification de gratuité pendant\n"
+       "    le listage des progiciels."
+msgid "msg_help_option_help"
+msgstr "    Afficher ce menu d'aide puis quitter"
+msgid "msg_help_option_list"
+msgstr "    En surcroît d'afficher un sommaire des progiciels gratuits et non-gratuits,"
```
Comment (suve) [inline lang/fr.po]: Missing linebreak (`\n`).

File: man/fr.man [hunk 3]
```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
```
Comment (suve) [inline man/fr.man]: All the section headings (`.SH`) should also be translated.

File: lang/fr.po [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+msgid "msg_help_option_version"
+msgstr "    Afficher la version puis quitter."
+msgid "msg_list_noargument"
+msgstr "%s: --list option requiert un argument."
+msgid "msg_list_badargument"
+msgstr "%s: l'argument pour --list option doit être 'none', 'free', 'nonfree' ou 'all'"
+msgid "msg_unknown_option"
+msgstr "%s: option inconnue '%s'"
+msgid "msg_total_free"
+msgstr "%d progiciels gratuits"
```
Comment (suve) [inline lang/fr.po]: With "Free Software", the "free" part is not about price (as in "free beer"), but about freedom. Taking a look at the [French Wikipedia](https://fr.wikipedia.org/wiki/Logiciel_libre), it uses "libre" and "propriétaire" for "free" and "proprietary".

File: lang/fr.po (hunk 1 above)
Comment (Philip-Nicolas) [reply lang/fr.po]: Yes.

File: lang/fr.po (hunk 4 above)
Comment (Philip-Nicolas) [reply lang/fr.po]: Absolutely! I knew the way I wrote it sounded weird, but I didn't know a better way to phrase it. Thanks!

Comment (suve) [reply lang/fr.po]: You seem to have missed this single occurence of "gratuits".  :)

**Comment by suve** [comment]
Thanks a ton! Merged.

**Pull request merged**